tests/test_evpn_vni.rb

```diff
@@ -0,0 +1,101 @@
+# Evpn Vni Unit Tests
+#
+# Andi Shen, December, 2015
+#
+# Copyright (c) 2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/evpn_vni'
+
+# TestEvpnVni - Minitest for EvpnVni class
+class TestEvpnVni < CiscoTestCase
+  def setup
+    # Disable feature bgp and no overlay evpn before each test to
+    # ensure we are starting with a clean slate for each test.
+    super
+    config('no feature bgp')
+    config('no nv overlay evpn')
+    config('no evpn')
+  end
+
+  def test_evpn_vni_create_and_destroy
```

Comment:
Please remove the `test_evpn_vni_` prefix from the test method names, e.g.:

```
  def test_create_and_destroy
  def test_route_distinguisher
  def test_route_target
```

 We're starting to clean up the test files and part of that is shortening the method names, which don't add any value since the test file already identifies the method.